### DISCUSSION TITLE
JWT Lazarus

### DISCUSSION THREAD

**Discussion by rodrigoferreiramonteiro** [HashLoad/horse#315] [Q&A] [answered]
Senhores é possivel gerar o Token no Lazarus?

Estou perguntando pois os exemplos contidos no git sao apenas para delphi, pois utilizam a unit delphi-jose-jwt

Se puderem fornecer um exemplo em lazarus agradeceria

**Comment by viniciussanchez** [comment, marked as answer]
Sim é possível. Aqui está os exemplo:
https://github.com/HashLoad/horse-jwt/tree/master/samples/lazarus

**Reply by rodrigoferreiramonteiro** [reply]
Boa Vinicius, nao tinha visto pq qdo efetuo a instalação via boss install horse-jwt

Por algum motivo na pasta SAMPLES dentro de modules ele esta colocando apenas o exemplo do "client" nao criando a pasta com exemplo do "auth"

Mas funcionou perfeito, tive que remover a unit DCP da uses pq nao estava encontrando

Mas o token foi gerado e validado

**Comment by andre-djsystem** [comment]
Veja este repositório - https://github.com/andre-djsystem/LazJWT
